doc/nrf/ug_thingy53.rst

```diff
@@ -106,57 +105,46 @@ Note that building for the non-secure application core is not fully supported.
 | nRF5340 SoC - Network core     |``thingy53_nrf5340_cpunet``                               |
 +--------------------------------+----------------------------------------------------------+
 
+.. note::
+   The |NCS| samples and applications that are compatible with Thingy:53 follow the :ref:`thingy53_app_guide`.
+   This is needed, because the samples must, among others, use consistent partition map and bootloader configuration.
```

Comment:
```suggestion
   This is needed, because the samples must use consistent partition map and bootloader configuration.
```
Unnecessary and adds too many commas to the line.